pkg/nuctl/command/delete.go

```diff
@@ -66,21 +67,26 @@ func newDeleteFunctionCommandeer(deleteCommandeer *deleteCommandeer) *deleteFunc
 		Short:   "Delete functions",
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			// if we got positional arguments
-			if len(args) != 1 {
+			if len(args) < 1 {
 				return errors.New("Function delete requires an identifier")
 			}
 
-			commandeer.functionConfig.Meta.Name = args[0]
-			commandeer.functionConfig.Meta.Namespace = deleteCommandeer.rootCommandeer.namespace
+			for argIndex, arg := range args {
```

Comment:
The iteration logic performed here must be accompanied with comments! You should always comment code you're adding (and update comments for code you're changing, rather than just delete them (like you did above)